--- /dev/fd/63	2025-06-05 00:05:40.536729599 -0400
+++ logging.go	2025-06-04 23:59:08.324388049 -0400
@@ -39,10 +39,11 @@
 
 // msglog provides package wide logging consistency for discordgo
 // the format, a...  portion this command follows that of fmt.Printf
-//   msgL   : LogLevel of the message
-//   caller : 1 + the number of callers away from the message source
-//   format : Printf style message format
-//   a ...  : comma separated list of values to pass
+//
+//	msgL   : LogLevel of the message
+//	caller : 1 + the number of callers away from the message source
+//	format : Printf style message format
+//	a ...  : comma separated list of values to pass
 func msglog(msgL, caller int, format string, a ...interface{}) {
 
 	if Logger != nil {
@@ -58,7 +59,12 @@
 		fns := strings.Split(name, ".")
 		name = fns[len(fns)-1]
 
-		msg := fmt.Sprintf(format, a...)
+		var msg string
+		if len(a) == 0 {
+			msg = fmt.Sprint(format)
+		} else {
+			msg = fmt.Sprintf(format, a...)
+		}
 
 		log.Printf("[DG%d] %s:%d:%s() %s\n", msgL, file, line, name, msg)
 	}
